process/smartContract/scQueryService.go

```diff
@@ -97,11 +97,6 @@ func (service *SCQueryService) executeScCall(query *process.SCQuery, gasPrice ui
 		}
 	}
 
-	err = service.checkVMOutput(vmOutput)
```

Comment:
i think that `checkVMOutput` function remains unsued

Reply:
done